docs/proposals/002-routing-processor-proposal/routing-processor.proto

```diff
@@ -0,0 +1,188 @@
+syntax = "proto3";
+
+package routing_processor.v1alpha1;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "envoy/service/ext_proc/v3/external_processor.proto";
```

Comment:
well, please do not expose/use the Envoy Extproc implementation detail at this layer - if you use that, then what's the point of abstraction we are introducing in this project vs you can setup your own extproc by yourself.